articles/quickstart/webapp/java-ee/_includes/_login.md

```diff
@@ -0,0 +1,379 @@
+## Configure Java EE Security
+
+The Java EE 8 Security API introduced the `HttpAuthenticationMechanism` interface to enable applications to obtain a user's credentials. Default implementations exist for Basic and form-based authentication, and it provides an easy way to configure a custom authentication strategy.
+
+To authenticate users with Auth0, you can implement the `HttpAuthenticationMechanism` interface and some of its collaborators. The key interfaces involved are:
```

Comment:
Collaborators seems an odd word choice and made me immediately think of human collaborators on a project...

Reply:
Yeah, that's not what we want readers to think :smile:

How about, "To authenticate users with Auth0, you can provide custom implementations of the following interfaces:"

Reply:
Updated.